src/plugins/data/public/antlr/opensearch_sql/grammar/OpenSearchSQLParser.g4

```diff
@@ -291,7 +291,7 @@ expression
 
 predicate
    : expressionAtom                                         # expressionAtomPredicate
-   | left = predicate comparisonOperator right = predicate  # binaryComparisonPredicate
+   | left = predicate comparisonOperator right = constant   # binaryComparisonPredicate
```

Comment:
why do we need this change?

Reply:
This line, along with line 307/308, was changed so that values don't get suggested for the left side of a predicate, while the right side doesn't have field names suggested and only values suggested instead. Although having a constant on the left and a field on the right does get parsed, suggesting it is practically useless/ information overload. There would be too many suggestions and it doesn't make sense to show every single possibility.